File: P3-ontologija-API/src/Svalina/Ontologija.php
<?php

namespace Svalina;

/**
 * @Entity @Table(name="ontologija")
 **/


class Ontologija 

{
    /** @id @Column(type="integer") @GeneratedValue **/
    protected $sifra;

    /**
    * @Column(type="string")
    */
    private $nazivFilma;

   /**
    * @Column(type="string")
    */
    private $zanr;

    /**
    * @Column(type="string")
    */
    private $nagrada;

    /**
    * @Column(type="integer")
    */
    private $godinaIzlaska;


    /**
    * @Column(type="string")
    */

    private $trajanjeFilma;



    

    public function getSifra(){
      return $this->sifra;
    }
  
    public function setSifra($sifra){
      $this->sifra = $sifra;
    }
  
    public function getNazivFilma(){
      return $this->nazivFilma;
    }
  
    public function setNazivFilma($nazivFilma){
      $this->nazivFilma = $nazivFilma;
    }
  
    public function getZanr(){
      return $this->zanr;
    }
  
    public function setZanr($zanr){
      $this->zanr = $zanr;
    }
  
    public function getNagrada(){
      return $this->nagrada;
    }
  
    public function setNagrada($nagrada){
      $this->nagrada = $nagrada;
    }
  
    public function getGodinaIzlaska(){
      return $this->godinaIzlaska;
    }
  
    public function setGodinaIzlaska($godinaIzlaska){
      $this->godinaIzlaska = $godinaIzlaska;
    }
  
    public function getTrajanjeFilma(){
      return $this->trajanjeFilma;
    }
  
    public function setTrajanjeFilma($trajanjeFilma){
      $this->trajanjeFilma = $trajanjeFilma;
    }

    

    


    public function setPodaci($podaci)
    {
      foreach($podaci as $kljuc => $vrijednost){
        $this->{$kljuc} = $vrijednost;
      }
    }
    
  }

?>

File: P3-android-app/app/src/main/java/ffos/p3/ontologija/Ontologija.java
package ffos.p3.ontologija;

import java.io.Serializable;

public class Ontologija implements Serializable {

    public Ontologija() {
        this.sifra = sifra;
        this.nazivFilma = nazivFilma;
        this.zanr = zanr;
        this.nagrada = nagrada;
        this.godinaIzlaska = godinaIzlaska;
        this.trajanjeFilma = trajanjeFilma;
    }

    private String nazivFilma;
    private String zanr;
    private String nagrada;
    private int godinaIzlaska;
    private String trajanjeFilma;

    private int sifra;

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

    public String getNazivFilma() {
        return nazivFilma;
    }

    public void setNazivFilma(String nazivFilma) {
        this.nazivFilma = nazivFilma;
    }

    public String getZanr() {
        return zanr;
    }

    public void setZanr(String zanr) {
        this.zanr = zanr;
    }

    public String getNagrada() {
        return nagrada;
    }

    public void setNagrada(String nagrada) {
        this.nagrada = nagrada;
    }

    public int getGodinaIzlaska() {
        return godinaIzlaska;
    }

    public void setGodinaIzlaska(int godinaIzlaska) {
        this.godinaIzlaska = godinaIzlaska;
    }

    public String getTrajanjeFilma() {
        return trajanjeFilma;
    }

    public void setTrajanjeFilma(String trajanjeFilma) {
        this.trajanjeFilma = trajanjeFilma;
    }


}
